Detectors/GlobalTracking/src/MatchGlobalFwd.cxx

```diff
@@ -306,7 +309,8 @@ void MatchGlobalFwd::loadMatches()
     auto MCHId = match.getMCHTrackID();
     LOG(debug) << "     ==> MFTId = " << MFTId << " MCHId =  " << MCHId << std::endl;
 
-    auto& thisMCHTrack = mMCHWork[MCHId];
+    auto& thisMCHTrack = mMCHWork[mMCHID2Work[MCHId]];
+    LOG(info) << " MFTId: " << MFTId << " MCHId: " << MCHId << " --> mMCHID2Work[MCHId]:" << mMCHID2Work[MCHId];
```

Comment:
I guess this is ready to merge, could you please just remove this info, it will produce lot of output.